django/forms/fields.py

```diff
@@ -90,11 +90,14 @@ def __init__(self, required=True, widget=None, label=None, initial=None,
         #                        hidden widget with initial value after widget.
         # validators -- List of additional validators to use
         # localize -- Boolean that specifies if the field should be localized.
+        # disabled -- Boolean that specifies whether the field is disabled, that
+        #             is its value is shown in the form but not editable.
```

Comment:
consider changing `value` = `Widget`, as the attr is just being passed down to the renderer in this case, isn't it?